### ISSUE TITLE
Button padding when using data-width="fit" is incorrect.

### ISSUE THREAD

**Issue by caseyjhol** [snapappointments/bootstrap-select-temp#17]
Originally reported by @mdczaplicki (https://github.com/silviomoreto/bootstrap-select/issues/1135#issuecomment-368252925):
> This is how it looks using BS4 and with these attributes: `htmlAttributes: new { @class = "selectpicker", multiple = "multiple", data_width = "fit", title = "Select roles" }`
Kinda too high I think. Also padding is broken IMO.
![image](https://user-images.githubusercontent.com/9108586/36634148-d6945b7e-19a8-11e8-8e3f-5842f8880076.png)
Removing "fit" makes it look good.
![image](https://user-images.githubusercontent.com/9108586/36634223-0163a408-19aa-11e8-9112-d9cfa7c5e6e3.png)

Demo: https://plnkr.co/edit/AYv6YSnrk5WwNgtSh6N0?p=preview

**Comment by krystyna93** [comment]
@caseyjhol Thanks for the solution to this.

@caseyjhol Is this the same solution that I can do if the 'tick' icon overlaps a dropdown option item that is too short? 

Thanks.

**Comment by caseyjhol** [comment]
@krystyna93 Can you create a new issue with details on how to recreate the overlap problem?

**Comment by krystyna93** [comment]
@caseyjhol this is the issue I am having trouble with. 
As you can see the first dropdown option is obscured by the tick icon, when the selectpicker has a data-width="fit" applied to it:
![2a9a61b5-32da-4bde-8b1a-b38ddba31f65](https://user-images.githubusercontent.com/22934714/37552237-43ce0be2-2a05-11e8-9ac8-b30196524a6c.png)
```
<select class="selectpicker" multiple title="Choose condiments" data-size="10" data-width="fit">
          <option>all ages</option>
          <option>18+</option>
          <option>17</option>
          <option>16</option>
          <option>15</option>
          <option>14</option>
</select>
```
I wish to keep the data-width="fit", for my layout purposes, so is there a way I can tweak the css for this?

**Comment by caseyjhol** [comment]
@krystyna93 Hrmmm...it's working properly for me (at least with the latest version in the dist folder on the v1.13.0-dev branch). https://plnkr.co/edit/g45uxyIeMcV0mRIl6NbN?p=preview. Are you sure it's not related to something with the CSS on your end?

**Comment by krystyna93** [comment]
@caseyjhol Ah okay. I tried using the new beta version, and it seems to be working now. Thanks.

**Comment by krystyna93** [comment]
@caseyjhol I have another issue now, that I cant seem why it would display like so:

The dropdown arrow is not positioned correctly...?
![93a14100-a4de-4e46-9691-ea487e192ee9](https://user-images.githubusercontent.com/22934714/37561676-99312f94-2aa8-11e8-84c6-7aee409c551c.png)

       <form>
          <div class="row">
             <div class="col-md-2">
               <div class="form-group">
                    <select class="selectpicker form-control" multiple title="All Ages">
                    <option>All Ages</option>
                    <option>18+</option>
                    <option>17</option>
                    <option>16</option>
                  </select>                  
            </div>
          </div>

          <div class="col-md-2">
              <div class="form-group">
                  <select class="selectpicker form-control" multiple title="All Ages">
                      <option>All Ages</option>
                      <option>18+</option>
                      <option>17</option>
                      <option>16</option>
                    </select>                  
              </div>
            </div>
        
        </div>  
      </form>

I have the beta version bootstrap-select.css and bootstrap-select.js with bootstrap v4.0.0

**Comment by caseyjhol** [comment]
Released in [v1.13.0](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.0)!
